Jenkinsfile

```diff
@@ -61,8 +64,19 @@ pipeline {
                     git lfs pull
                     git lfs checkout
                     cd $curdir
+                    export api_commit_sha=$(git submodule status | grep 'pmm-api-tests' | awk -F ' ' '{print $1}')
+                    export api_branch=$(git config -f .gitmodules submodule.pmm-api-tests.branch)
```

Comment:
Our APIs are defined in `pmm` repo. Let's use a longer name for API tests. Like `apt_tests_branch`, etc.